### ISSUE TITLE
Encoder network wont let me update get error now about mysqli

### ISSUE THREAD

**Issue by JoshWho** [WWBN/AVideo#6270]
```
root@joshwhotv:/var/www/html/YouPHPTube/Uploader/install# php encoders.php

Warning: Undefined array key "SERVER_NAME" in /var/www/html/YouPHPTube/Uploader/configuration.php on line 7

Fatal error: Uncaught Error: Class "mysqli" not found in /var/www/html/YouPHPTube/Uploader/objects/include_config.php:7
Stack trace:
#0 /var/www/html/YouPHPTube/Uploader/configuration.php(22): require_once()
#1 /var/www/html/YouPHPTube/Uploader/install/encoders.php(17): require_once('/var/www/html/Y...')
#2 {main}
  thrown in /var/www/html/YouPHPTube/Uploader/objects/include_config.php on line 7
root@joshwhotv:/var/www/html/YouPHPTube/Uploader/install# cd ..
root@joshwhotv:/var/www/html/YouPHPTube/Uploader# git pull
Already up to date.

```

**Comment by DanielnetoDotCom** [comment]
`Class "mysqli" not found ` is a php installation error. try to google for that, you will find the missing library

**Comment by JoshWho** [comment]
I have

php7.3-cli php7.3-common php7.3-json php7.3-opcache php7.3-readline php7.3-curl php7.3-gd php7.3-xml

**Comment by DanielnetoDotCom** [comment]
Looks like you are missing the php-MySQL .

again I strongly recommend you to google for that

**Comment by JoshWho** [comment]
php7.3-mysql is already the newest version (7.3.33-1+ubuntu18.04.1+deb.sury.org+1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

**Comment by JoshWho** [comment]
Weird part is whe I use command  php --version it shows 

```
PHP 8.1.2 (cli) (built: Jan 24 2022 10:42:15) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.2, Copyright (c) Zend Technologies

```

**Comment by JoshWho** [comment]
I fixed it some how by forcing the php 7.4 cli to work with it
